### PR TITLE
Compile in Linux without changing the pom.xml

### PR DESCRIPTION
**Pull request by enexusde**
See in-line comment.

### CODE REVIEW

**Comment by Geolykt** [comment]
The better question really is - why is that line there in the first place? I've always removed it outright whenever I wanted or needed to compile janino from source and it seems to work. Plus, this configuration will not even work on almost all windows setups as the path is extremely specific; using something like `${java.home}/bin/javac.exe` would be less specific, but would mean you'd loose control over which JDK is enacting the task. Since javadoc-maven-plugin 2.5 we could even use `${java.home}/bin` and make it work on both linux and windows.

This PR would be related to following commit - unfortunately nothing can be learned from that change as it remains vague in nature:
- https://github.com/janino-compiler/janino/commit/3c05045a382449343d7391afdb388c57b220d187 (Version 3.1.7)


TL;DR: I propose that we either drop the explicit definition of the javadoc executable altogether or to use `${java.home}/bin` as the default. One would then easily be able to override that using their user-wide maven-settings.xml if need be (without committing this user-specific configuration to the whole world).

**Comment by enexusde** [comment]
I do not insist. It reveals the PC's condition and environment (root-access, filestructure and operation-system) but I do not think a hacker could abuse that information. Feel free to do a review and add your suggestions.

**Comment by Geolykt** [comment]
What exactly do you mean with that? Especially the PC's condition and environment aspect I don't quite understand.

**Comment by enexusde** [comment]
Forget about it.

**Comment by aunkrig** [comment]
I probably set the javadocExecutable to enforce that the Javadoc is generated with JDK 1.8, i.e. in Javadoc 1.8 style with frames and stuff.
But certainly it's ugly to have an absolute path in the POM. It was a quick hack.
I’ll look into that.
